packages/paste-core/components/card/src/index.tsx

```diff
@@ -28,15 +35,15 @@ const Card = React.forwardRef<HTMLElement, CardProps>(
   }
 );
 
-if (process.env.NODE_ENV === 'development') {
-  Card.propTypes = {
-    padding: isPaddingTokenProp,
-    paddingTop: isPaddingTokenProp,
-    paddingRight: isPaddingTokenProp,
-    paddingBottom: isPaddingTokenProp,
-    paddingLeft: isPaddingTokenProp,
-  };
-}
+Card.propTypes = {
+  /** element identifier for customization */
```

Comment:
NIT: Could this comment go up above by the type definition? I am not sure how we could do this, but it would be nice if the `.d.ts` file could have this comment to clarify the purpose of `element`.

Reply:
Yeah I'm into it.